src/tools/genpolicy/rules.rego

```diff
@@ -1633,6 +1633,31 @@ ExecProcessRequest {
     print("ExecProcessRequest 3: true")
 }
 
+UpdateInterfaceRequest {
+    print("UpdateInterfaceRequest 1: input =", input)
+
+    i_interface = input.interface
+    p_interface = policy_data.request_defaults.UpdateInterfaceRequest.interface
+
+    print("UpdateInterfaceRequest 1: p_interface.device =", p_interface.device)
+    print("UpdateInterfaceRequest 1: i_interface.device =", i_interface.device)
+
+    i_interface.device == p_interface.device
+
+    print("UpdateInterfaceRequest 1 device check passed")
+
+    i_interface = input.interface
+    p_interface = policy_data.request_defaults.UpdateInterfaceRequest.interface
+
+    print("UpdateInterfaceRequest 1: p_interface.IPAddresses =", p_interface.IPAddresses)
+    every ip in i_interface.IPAddresses {
+        print("UpdateInterfaceRequest 1: i_interface.IPAddresses[x] =", ip)
+        ip in p_interface.IPAddresses
+    }
+
+    print("UpdateInterfaceRequest 1: true")
```

Comment:
Should we enforce the interface name as well? per https://microsoft.visualstudio.com/OS/_workitems/edit/45722553#104714911

Reply:
The mask is included in the IPAddresses and is compared as part of that check.